examples/api/lib/animation/animation_controller/animated_digit.0.dart

```diff
@@ -191,16 +195,12 @@ class _AnimatedDigitHomeState extends State<AnimatedDigitHome> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
-        child: AnimatedDigit(value: value % 10),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: () {
-          setState(() { value += 1; });
-        },
-        tooltip: 'Increment Digit',
-        child: const Icon(Icons.add),
-      ),
+      body: Center(child: AnimatedDigit(value: value % 10)),
+      floatingActionButton: FloatingActionButton(onPressed: () {
+        setState(() {
+          value += 1;
+        });
+      }, tooltip: 'Increment Digit', child: const Icon(Icons.add)),
```

Comment:
This feels a step backwards and makes it hard to scan what the child is, or what properties on FloatingActionButton are even set. It looses the tree structure of the widget tree.

Reply:
Yeah, agreed that doesn't look good. I'll add this as an example on https://github.com/dart-lang/dart_style/issues/1527.